README.rst

```diff
@@ -0,0 +1,152 @@
+Elm Docstyle
+============
+A tool that allows you to check the existence and quality of your `Elm <http://elm-lang.org/>`_ code comments.
+
+Usage
+=====
+
+Prerequisites
+
+
+The following binaries should be available on the path:
+
+
+.. code-block:: bash
+
+    node >=6
+    elm  0.18.x
+
+
+Installation
+============
+
+With ``npm``:
+
+.. code-block:: bash
+
+    npm install -g elm-docstyle
+
+or if using ``yarn``:
+
+
+.. code-block:: bash
+
+    yarn global add elm-docstyle
+
+
+Usage
+=====
+
+``elm-docstyle [elm_code_directory | path_to_elm_file]``
+
+Options:
+    --help, -h          Print the help output.
+    --version, -v       Print the package version.
+    --verbose           If set, the offending comments are added to the error report in full.
+    --check_all         If set, the checker goes through all declarations; otherwise, only through exported ones.
```

Comment:
I'd clarify a bit (can you have multi-line descriptions with new lines?):
```
If set, all the function declarations are checked (including the non-exported ones). 

Otherwise, only the exported function declarations are checked.
```